clients/src/main/java/org/apache/kafka/clients/consumer/StickyAssignor.java

```diff
@@ -186,15 +186,16 @@
     private static final Schema STICKY_ASSIGNOR_USER_DATA = new Schema(
             new Field(TOPIC_PARTITIONS_KEY_NAME, new ArrayOf(TOPIC_ASSIGNMENT)));
 
-    Map<String, List<TopicPartition>> currentAssignment = new HashMap<>();
+    Map<String, List<TopicPartition>> currentAssignment;
```

Comment:
Hmm.. if we're not caching the previous assignment, why do we still need this field?